src/modules/filters/components/FilterBar/FilterBar.jsx

```diff
@@ -12,74 +13,68 @@ const { Option } = Select;
 const FilterBar = () => {
     const { styles } = useFelaEnhanced(felaRules);
     const intl = useIntl();
+
+    const { fetchReports, inProgress } = useReportsFetcher();
     const { appName, platform, deviceModel } = useFilters();
+
+    const [form] = Form.useForm();
+
+    const handleChange = form => {
+        fetchReports(form.getFieldsValue());
+    };
+    const filterProps = [
+        {
+            name: 'appName',
+            options: ['All'].concat(appName),
+            label: 'App name',
```

Comment:
This props should be also in translations sheets

Reply:
done